src/linker/Linker/Position.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Mono.Linker
+{
+	public class Position
```

Comment:
I would prefer a bit more specific name - maybe `MessageSourcePosition`

Reply:
Is the extra type actually helpful, why not just have Line, Column in MessageOrigin?